datamodel/app/create_app.py

```diff
@@ -53,13 +53,14 @@ def create_app(
     """
     cwd = Path(__file__).parent.resolve()
     variables = {
-        "SRID": psycopg.sql.SQL(f"{srid}")
+        "SRID": psycopg.sql.SQL(f"{srid}"),
     }  # when dropping psycopg2 support, we can use the srid var directly
 
     if drop_schema:
         run_sql("DROP SCHEMA IF EXISTS tww_app CASCADE;", pg_service)
-
-    run_sql("CREATE SCHEMA tww_app;", pg_service)
+        run_sql("CREATE SCHEMA tww_app;", pg_service)
+    else:
```

Comment:
why do we need to allow working on an existing schema?

Reply:
I agree, not strictly necessary. Would you remove the drop_schema flag entirely?

Reply:
I would still keep it, I found it useful when developping

Reply:
We could always drop and re-create